.sandbox/provision/lifecycle-asg/common.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+function redirect_stdout() {
+    exec 3>&1
+    exec 1>&2
+}
+
+function restore_stdout() {
+    exec 1>&3
+    exec 3>&-
+}
+
+function get_volume_id() {
+    volume_info="$(aws ec2 describe-volumes --filters Name=tag:SandboxID,Values=$SANDBOX_ID)"
+    jq -r '.Volumes[0].VolumeId' <<< $volume_info
```

Comment:
As the convention of bash script, please use quote around variable dereferencing all the time, unless there's special usage/reason. This includes variable deref `"$NAME"`, `"${NAME}"` etc, and subcommand `"$(command)"`.